packetery/template/email/footer.latte

```diff
@@ -0,0 +1,19 @@
+{varType \Packetery\Order\Wrapper $order}
+
+<h2>
+    {__( 'Shipping Detail', 'packetery' )}
+</h2>
+<table class="td" style="border-spacing: 0; border: 0; vertical-align: middle; width: 100%;">
```

Comment:
styly prosím do css souboru.

Reply:
WC email šablona to nedělá. A je k tomu důvod. https://stackoverflow.com/a/2105975